x-pack/plugin/core/src/main/resources/logs-default-settings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "template": {
+    "settings": {
+      "index": {
+        "lifecycle": {
+          "name": "logs-default-policy"
+        },
+        "codec": "best_compression",
```

Comment:
Any recommendations here?

Reply:
This is a throughput-storage tradeoff, so the question is just which one we want to default to. This is what the current templates were, but with our other data tiering plans (searchable snapshots), maybe it doesn't make sense any more, I'll add it to the "to discuss" pile

Reply:
One thing that crossed my mind recently is whether the templates are the right way to configure this default, or whether this would be a node-level default, e.g. we could default to `best_compression` on the warm nodes.